pycobertura/cobertura.py

```diff
@@ -58,6 +67,17 @@ def __init__(self, report, filesystem=None):
     def __eq__(self, other):
         return self.report and other.report and self.report == other.report
 
+    def _load_from_path(self, report_path):
+        # it's not a xml-string, so it should be a path.
+        # is it existed?
+        if not os.path.isfile(report_path):
+            raise self.InvalidCoverageReport("file not existed: {}".format(report_path))
```

Comment:
Can we tweak the wording?

```suggestion
            raise self.InvalidCoverageReport("Invalid coverage file: {}".format(report_path))
```